ui/src/index.tsx

```diff
@@ -460,6 +460,15 @@ class Root extends PureComponent {
                             path="checks/:checkID"
                             component={CheckHistory}
                           />
+                          <Route path="about" component={OrgProfilePage}>
+                            <Route path="rename" component={RenameOrgOverlay} />
+                          </Route>
+                          <Route path="about" component={OrgProfilePage}>
+                            <Route path="rename" component={RenameOrgOverlay} />
+                          </Route>
```

Comment:
I needed two different routes for the same page because in `treeNav` mode `about` has a different route than with the existing nav

Reply:
I'm so confused. Aren't these the same routes?

Reply:
Currently the `about` page is located at `orgs/orgid/settings/about` but in the new tree nav it is `orgs/orgid/about`. Since both can exist I can't remove the old route just yet